x-pack/plugins/infra/public/pages/metrics/hosts/components/hosts_list_table.tsx

```diff
@@ -0,0 +1,300 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+import { TypedLensByValueInput } from '@kbn/lens-plugin/public';
+import type { Query, TimeRange } from '@kbn/es-query';
+import React from 'react';
+import type { DataView } from '@kbn/data-views-plugin/public';
+import { InfraClientStartDeps } from '../../../../types';
+
+const getLensHostsTable = (
+  metricsDataView: DataView,
+  query: Query
+): TypedLensByValueInput['attributes'] =>
+  ({
+    visualizationType: 'lnsDatatable',
```

Comment:
This is cool.

Did you find a nice way to get from a lens viz to this code or was there a lot of formatting/replacement involved?

Reply:
I created the table in Lens and then used the "Open in Playground" feature that gives you all the attributes  and just basically copied it into here https://github.com/elastic/kibana/tree/main/x-pack/plugins/lens#getting-started.  Vice versa would work as well.  I didn't have to do any direct editing of this code other than handing it a Data View.  Everything was done in the Lens UI.  I was telling Milton in another comment (https://github.com/elastic/kibana/pull/138173#discussion_r953736958) that the idea would be we'd store this data somewhere as models and generate the lens attributes ourselves once we've got most of the data from the Lens Playground.  You can also always paste this back into Lens and view your viz and edit it there.